tests/fs/test_nodefs_cloexec.c

```diff
@@ -7,11 +7,14 @@
 int main(void)
 {
     EM_ASM(
+#ifndef NODERAWFS
         FS.mkdir('/working');
         FS.mount(NODEFS, {root: '.'}, '/working');
-        FS.close(FS.open('/working/test.txt', 'w'));
+        FS.currentPath = '/working';
+#endif
+        FS.close(FS.open('test.txt', 'w'));
     );
-    assert(open("/working/test.txt", O_RDONLY | O_CLOEXEC) != -1);
+    assert(open("test.txt", O_RDONLY | O_CLOEXEC) != -1);
```

Comment:
The removal of `/working/` could be seen as changing the test, although searching through the tests/ subdirectory, it does look like there's still a couple of places where we explicitly test the `/working/test.txt` form. I think we'd like to ensure that this is tested, since this does change absolute to relative, which sometimes has different behavior. Perhaps a

```c
#ifdef NODERAWFS
#define ROOT ""
#else
#define ROOT "/"
#endif

...

assert(open(ROOT "working/test.txt", O_RDONLY | O_CLOEXEC) != -1);

```

or similar, to keeping testing behavior of absolute paths, while retaining the relative test for NODERAWFS?

(I'm not sure how important specifically the absolute root path property was when the original test was being crafted, but I think it's good to keep it as close to original as possible, perhaps only adding new cases)